### DISCUSSION TITLE
Github:I try to visit https://github.com/BiliZBro/bilizbro.github.io "Site not found" error occurs

### DISCUSSION THREAD

**Discussion by orangeZSCB** [community/community#29330] [Other Feature Feedback, Questions, & Ideas]
I try to visit https://github.com/BiliZBro/bilizbro.github.io "Site not found" error occurs
Is this a GitHub pages bug?
(i'm chinese, translate by Baidu)

**Comment by melvincwng** [comment]
@BiliZBro, can you try to clear your browse cache? Could be a caching issue.

I'm able to visit that link which you mentioned in your post earlier.

**Reply by orangeZSCB** [reply]
![image](https://user-images.githubusercontent.com/90908765/184472745-2ac3499a-0208-419b-a320-3bd3bd1aa47b.png)
oh holy……

**Reply by melvincwng** [reply]
@BiliZBro, can you try this link?

https://github.com/BiliZBro/BiliZBro.github.io

It could be case-sensitive.

**Reply by orangeZSCB** [reply]
no
![image](https://user-images.githubusercontent.com/90908765/184473672-3d4813a4-103d-4eda-bf91-4c85cdb31969.png)

**Reply by melvincwng** [reply]
@BiliZBro , alternatively can you try in another browser? Because on my end, it's working fine.

Could it also be that your internet traffic is intercepted/blocked by your Internet Service Provider? 

Hence, resulting in you being unable to visit that github repo's page.

**Reply by orangeZSCB** [reply]
Another browser tried
maybe is blocked by my Internet Service Provider
i'll ask
